### PR TITLE
Add language filter for Rust (#168)

### PR DESCRIPTION
**Pull request by whimboo**
This PR will fix issue #168. @jdm can you please have a look? Thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,9 +129,17 @@
...
-                       ["js", "JavaScript"], ["cpp", "C/C++"], ["html", "HTML/CSS"], ["xml", "XML/XUL"],
-                       ["perl", "Perl"]]);
+        addCategories([
+          ["cpp", "C/C++"],
+          ["html", "HTML/CSS"],
+          ["java", "Java"],
+          ["js", "JavaScript"],
+          ["perl", "Perl"],
+          ["py", "Python"],
+          ["rs", "Rust"],
```
Comment (jdm) [inline index.html]: Don't you need to add something to the list at https://github.com/whimboo/bugsahoy/blob/ea3e94b0957eafb4cd0917bf920c91db295f2280/magic.js#L348 as well? Does this return any results when you test it locally right now?

**Comment by jdm** [comment]
Also, I would rather use the `lang=rust` whiteboard tag, which is a bit more intuitive than `lang=rs` in my opinion. That means we should update https://bugzilla.mozilla.org/show_bug.cgi?id=1399158.

File: index.html (hunk 1 above)
Comment (whimboo) [reply index.html]: Oh, you are right. I searched the code for `py` to find other places which needs to be updated but didn't find any. Strange that there is. Adding such a line clearly makes the bug appear.

**Comment by whimboo** [comment]
The updated code works fine for me locally now.

**Comment by jdm** [comment]
Thanks!

**Pull request merged**